common/src/commonMain/kotlin/org/hisp/dhis/common/screens/Components.kt

```diff
@@ -1,17 +1,18 @@
 package org.hisp.dhis.common.screens
 
-enum class Components {
-    HOME,
-    ICON_BUTTON,
-    BUTTON,
-    FLOATING_BUTTON,
-    CARD,
-    CHIPS,
-    DIALOGS,
-    RADIO,
-    SWITCH,
-    SLIDER,
-    SNACKBAR,
-    TABS,
-    TEXTFIELD
+enum class Components(val label: String) {
```

Comment:
these items aren't the real ones, they where a sample, please remove unused ones